rdfunit-junit/src/main/java/org/aksw/rdfunit/junit/RdfUnitJunitStatusTestExecutor.java

```diff
@@ -0,0 +1,36 @@
+package org.aksw.rdfunit.junit;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+
+import org.aksw.rdfunit.exceptions.TestCaseExecutionException;
+import org.aksw.rdfunit.io.reader.RDFModelReader;
+import org.aksw.rdfunit.sources.TestSource;
+import org.aksw.rdfunit.sources.TestSourceBuilder;
+import org.aksw.rdfunit.tests.executors.RLOGTestExecutor;
+import org.aksw.rdfunit.tests.query_generation.QueryGenerationSelectFactory;
+import org.aksw.rdfunit.tests.results.TestCaseResult;
+
+final class RdfUnitJunitStatusTestExecutor extends RLOGTestExecutor {
+
+    public RdfUnitJunitStatusTestExecutor() {
+        super(new QueryGenerationSelectFactory());
+    }
+
+    Collection<TestCaseResult> runTest(RdfUnitJunitTestCase rdfUnitJunitTestCase)
+            throws IllegalAccessException, InvocationTargetException {
+        final TestSource modelSource = new TestSourceBuilder()
+                .setPrefixUri("custom", "rdfunit")
```

Comment:
@jimkont Could you explain what's the thing with the Prefix URI? Just noticed we just define constant / hard coded values to satifsfy the Builder.. However not sure if this is correct.